srcpkgs/bees/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'bees'
+pkgname=bees
+version=081a6af278fc2b4f33198e2ec0028f8383bb6d4a
+revision=1
+build_style=gnu-makefile
+makedepends="gcc git discount"
```

Comment:
Should be hostmakedepends, furthermore GCC is always there, no need to list it.

Reply:
Thank you, I will make those changes now.